lib/Traits/TraitsClasses/CkanCatalogGroupTraits.ts

```diff
@@ -60,4 +60,11 @@ export default class CkanCatalogGroupTraits extends mixTraits(
     description: `True to allow entire WMS servers (that is, WMS resources without a clearly-defined layer) to be added to the catalog; otherwise, false.`
   })
   allowEntireWmsServers: boolean = true;
+
+  @primitiveTrait({
+    type: "boolean",
+    name: "Exclude inactive datasets",
+    description: `True to remove inactive datasets (\`state\` is not \`active\` OR )\`data_state\` is not \`active\`.`
```

Comment:
How come the backticks around the entire description? Should I be doing this?

Reply:
Nah I don't know why I did that. I have changed to double quotes.

Reply:
Sometimes I prefer backticks if I have nested quotes and things in the description - as the field is rendered as markdown in our docs